README.md

```diff
@@ -332,7 +332,7 @@ Read more → [Customizing the location hook](#customizing-the-location-hook).
 - **`matcher: (pattern: string, path: string) => [match: boolean, params: object]`** — a custom function used for matching the current location against the user-defined patterns like `/app/users/:id`. Should return a match result and an hash of extracted parameters. It should return `[false, null]` when there is no match.
 
 - **`base: string`** — an optional setting that allows to specify a base path, such as `/app`. All application routes
-  will be relative to that path.
```

Comment:
Here I tried to be as succinct as possible to match the style of the ReadMe. I can elaborate a bit (maybe with samples?) as this might not be clear enough.